src/js/utils/dom.js

```diff
@@ -56,6 +56,15 @@ function classRegExp(className) {
 }
 
 /**
+ * Whether the current DOM interface appears to be real.
+ *
+ * @return {Boolean}
+ */
+export function isReal() {
+  return document === window.document && typeof document.createElement === 'function';
```

Comment:
since we're using `global`, it returns either window or global depending on the environment, so, `window` should never end up being undefined.